index.js

```diff
@@ -191,12 +191,29 @@ FileList.prototype = new (function () {
    * pattern for finding file to include in the list. Arguments should be strings
    * for either a glob-pattern or a specific file-name, or an array of them
    */
-  this.include = function (items) {
-    if (items) {
-      this.pendingAdd = this.pendingAdd.concat(items).filter(function (item) {
-        return !!item;
-      });
+  this.include = function () {
+    var args = arguments
+        , arg
+        , includes = { items: [], options: {} };
+
+    for (var i = 0, ilen = args.length; i < ilen; i++) {
+      arg = args[i]
+
+      if (typeof arg === 'object' && !Array.isArray(arg)) {
+        Object.assign(includes.options, arg)
```

Comment:
Not sure if we want to assume everyone has access to `Object.assign` -- but I might also be showing my age here. :)